lib/msprime.c

```diff
@@ -2537,8 +2537,10 @@ msp_run_dtwf(msp_t *self, double max_time, unsigned long max_events)
     unsigned long events = 0;
     int mig_source_pop, mig_dest_pop;
     sampling_event_t *se;
-    double mu;
-    uint32_t j, k, i, n;
+    uint32_t j, k, i, N, num_migs, max_migs;
+    unsigned int n[self->num_populations];
```

Comment:
This won't work on a lot of compilers I'm afraid (probably why the windows build is failing), and is a bit dodgy in any case. If you had a lot of populations you could overflow the stack frame, which would be a bad time for all. Best to just malloc/free the memory for now.